cl/assets/templates/base.html

```diff
@@ -252,7 +252,10 @@ <h1>You did not supply the "private" variable to your template.
               </a>
               <ul class="dropdown-menu" role="menu">
                 <li>
-                  <a href="{% url "advanced_o" %}" tabindex="101">Advanced Search</a>
+                  <a href="{% url "advanced_o" %}" tabindex="101">Opinions Search</a>
+                </li>
+				        <li>
+                  <a href="{% url "es_search" "parenthetical" %}" tabindex="101">Parentheticals Search</a>
```

Comment:
Let's put this feature behind a waffle so most people can't see it until we're happy. Even after we're happy with it, we won't want it shown by default until it is loaded up with data, which could take a while.

Reply:
Done, now is behind a waffle flag, we'll need to create the flag in prod via the admin.

Reply:
Flag added here: https://www.courtlistener.com/admin/waffle/flag/3/change/